nengo_viz/static/viz_sim_control.js

```diff
@@ -170,6 +154,13 @@ VIZ.TimeSlider = function(args) {
     /** The SimControl object */
     this.sim = args.sim;
 
+    /** get reference to the overall div */
+    this.div = $(".time_slider")[0];
```

Comment:
Sorry, I forgot to click the little expand-code button, so I thought this was in `VIZ.SimControl.prototype.on_resize`.